debugtools/DDR_VM/src/com/ibm/j9ddr/vm29/tools/ddrinteractive/commands/J9BCUtil.java

```diff
@@ -695,6 +696,8 @@ private static void dumpRomStaticField(PrintStream out, J9ROMFieldShapePointer r
 	 * Dump a printed representation of the specified @accessFlags to stdout.
 	 * Answer zero on success
 	 */
+	private static long J9AccClassIsUnmodifiableBit = J9ConstantHelper.getLong(J9JavaAccessFlags.class, "J9AccClassIsUnmodifiable", 0);
```

Comment:
This field should probably be `final` and separated from the unrelated comment above.

Reply:
final is fine, but I'm not sure anything is improved by moving the constant elsewhere (the top of the file would be the obvious choice).

Reply:
Or are you just asking that it be moved above the comment?

Reply:
Above the comment would be fine.

Reply:
The comment is stale: it talks about an 'answer' from a method returning `void`. Someday that might get fixed. ;-)

Reply:
I'll fix it while I'm here.